jrnl/editor.py

```diff
@@ -5,14 +5,18 @@
 import sys
 import tempfile
 import textwrap
+from pathlib import Path
 
 from .color import ERROR_COLOR
 from .color import RESET_COLOR
 from .os_compat import on_windows
 
 
-def get_text_from_editor(config, template=""):
-    filehandle, tmpfile = tempfile.mkstemp(prefix="jrnl", text=True, suffix=".txt")
+def get_text_from_editor(config, template="", suffix=".jrnl"):
```

Comment:
Small nitpick: since `suffix` is never passed as an argument to this function, can we move it into the function itself as a local variable? This will avoid confusion about the function signature.

Otherwise, this PR is great. Thank you!

Reply:
Oh, ofcourse! Let me do that right now.

Reply:
Done!